assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * DashboardUniqueVisitorsWidget Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME } from '../../datastore/constants';
+import { provideModules, provideSiteInfo } from '../../../../../../tests/js/utils';
+import { provideAnalyticsMockReport } from '../../util/data-mock';
+import { withWidgetSlug } from '../../../../googlesitekit/widgets/util/';
+import WidgetReportError from '../../../../googlesitekit/widgets/components/WidgetReportError';
+import WidgetReportZero from '../../../../googlesitekit/widgets/components/WidgetReportZero';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import DashboardUniqueVisitorsWidget, { selectSparklineArgs, selectReportArgs } from './DashboardUniqueVisitorsWidget';
+
+const currentEntityURL = 'https://www.example.com/example-page/';
+
+const Template = ( args ) => {
+	const widgetSlug = 'dashboardUniqueVisitorsWidget';
+
+	return (
+		<WithRegistrySetup func={ args.setupRegistry }>
+			<div className="googlesitekit-widget">
+				<div className="googlesitekit-widget__body">
+					<DashboardUniqueVisitorsWidget
+						WidgetReportError={ withWidgetSlug( widgetSlug )( WidgetReportError ) }
+						WidgetReportZero={ withWidgetSlug( widgetSlug )( WidgetReportZero ) }
```

Comment:
Let's wrap `DashboardUniqueVisitorsWidget` with `withWidgetComponentProps` instead which provides all of the expected component props 👍

Reply:
👍